samples/greeter-function.yaml

```diff
@@ -3,10 +3,10 @@ kind: Function
 metadata:
   name: greeter
 spec:
-  image: sk8s/java-function-invoker:v0001
+  image: sk8s/greeter:v0001
   protocol: http
   input: names
   output: greetings
   env:
   - name: FUNCTION_URI
-    value: file:///resources/java/greeter/target/greeter-1.0.0.jar?functions.Greeter
+    value: file:///functions/java/greeter-1.0.0.jar?functions.Greeter
```

Comment:
defaults would be nice here as well, but it's a bit trickier given that the function name might need to be specified... however, in the case where there's a single JAR and a single Function implementation in that JAR, the `java-function-invoker` could probably introspect the single JAR dropped into `/function`.